examples/todo-mvc-svelte/tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+  "extends": "@tsconfig/svelte/tsconfig.json",
+  "compilerOptions": {
+    "target": "esnext",
+    "module": "esnext",
+    "resolveJsonModule": true,
+    "baseUrl": ".",
+    /**
+     * Typecheck JS in `.svelte` and `.js` files by default.
+     * Disable checkJs if you'd like to use dynamic types in JS.
+     * Note that setting allowJs false does not prevent the use
+     * of JS in `.svelte` files.
+     */
+    "allowJs": true,
+    "checkJs": false
```

Comment:
This example is a weak-TypeScript project, because Svelte does not provide TypeScript support in the markdown https://github.com/sveltejs/svelte/issues/4701. But I wanted to keep the event handlers in the markup in `Todo.svelte` and `Todos.svelte`, so that the example remain tidy and compact.

Setting `chekcJs` to `false` allows both TypeScript and JavaScript for `.svelte` files, controlling with `<script lang="ts">` and `<script>` respectively.